packages/databyss-ui/modules/Sidebar/Sidebar.js

```diff
@@ -30,14 +30,19 @@ Section.defaultProps = {
   variant: 'heading3',
 }
 
+const padding = 26
+const headerHeight = 34
+const footerHeight = 48
+const searchBar = 54
+
+export const sidebarListHeight = `calc(100vh - ${pxUnits(
```

Comment:
I see why we're doing this, but we should make this more responsive. I think we should use flexbox to expand the navbar and sidebar vertically rather than positioning the new page buttons absolutely. Then we can let the flex layout do the work of keeping everything flush and aligned. We can do this in a separate PR if you like.

Reply:
Yeah I tried a couple of times to refactor this part but it's pretty difficult because the new page button is part of <SidebarCollpased> but it also needs to cover the <Sidebar> component, so I don't know if that's possible without position absolute. I think it probably makes sense to try in another PR...